inst/rmarkdown/templates/transmissibility-report/skeleton/rmdchunks/EpiNow2.Rmd

```diff
@@ -9,37 +9,55 @@ pacman::p_load("EpiNow2")
 ### Results
 
 ```{r}
-# TODO: remove harcoded SARS-CoV-2 params
-generation_time <- get_generation_time(
-  disease = "SARS-CoV-2",
-  source = "ganyani"
+# Approximate serial interval with gamma distribution since this is what EpiNow2
+# will use for generation_time
+si_gamma <- epiparameter::extract_param(
+  type = "range",
+  values = c(median(si$r(1e3)), min(si$r(1e3)), max(si$r(1e3))),
+  distribution = "gamma",
+  samples = 1e3
 )
-incubation_period <- get_incubation_period(
-  disease = "SARS-CoV-2",
-  source = "lauer"
-)
-reporting_delay <- list(
-  mean = convert_to_logmean(3, 1),
-  mean_sd = 0.1,
-  sd = convert_to_logsd(3, 1),
-  sd_sd = 0.1,
-  max = 10
+si_gamma <- epiparameter::gamma_shapescale2meansd(si_gamma[[1]], si_gamma[[2]])
+
+generation_time <- list(
+  mean = si_gamma$mean,
+  mean_sd = 0,
+  sd = si_gamma$sd,
+  sd_sd = 0,
+  max = max(si_x)
 )
 ```
 
 #### Global transmissibility
 
+These analyses present results for the global incidence, i.e. without
+stratification. Results include:
+ 
+* a graph of the estimates of $R_t$ over time, with associated 95% credibility
+  intervals
+  
+* a table summarising these results for the last two weeks
+
 ```{r}
-rt_epinow2 <- i_recent %>%
+dat_i_day <- dat_raw %>%
+  incidence("date",
+    interval = 1L,
+    counts = {{ count_var }},
+    groups = {{ group_var }}
+  )
+
+# Warning, this step will take 2 cores of your computer and may be running
+# for a long time!
+rt_epinow2 <- dat_i_day %>%
   regroup() %>%
   dplyr::rename(
     confirm = .data[[count_var]],
     date    = date_index
   ) %>%
   epinow(
     generation_time = generation_time,
-    delays = delay_opts(incubation_period, reporting_delay),
-    stan = stan_opts(samples = 10, chains = 1),
+    stan = stan_opts(samples = 1e3, chains = 2),
```

Comment:
have you set it to use multiple cores somewhere?

Reply:
No but it's still running with 2 cores. I had assumed you automatically set it up this way but if this is not unexpected, let me try to investigate what's happening.

Reply:
Its likely the mc.cores settings you have in your options. `stan` uses this if not told otherwise. Worth setting explicitly to make it clear to users what is happening when their computer explodes.